src/Microsoft.VisualStudio.Extensibility.Testing.Xunit.Shared/Harness/IdeTestAssemblyRunner.cs

```diff
@@ -267,9 +267,14 @@ protected override async Task<RunSummary> RunTestCollectionAsync(IMessageBus mes
                     executionMessageSinkFilter = new IpcMessageSink(ExecutionMessageSink, knownTestCasesByUniqueId, finalAttempt, completedTestCaseIds, cancellationTokenSource.Token);
                     marshalledObjects.Add(executionMessageSinkFilter);
 
+                    var environmentVariables = visualStudioInstanceKey.EnvironmentVariables.ToImmutableDictionary(
+                        variable => variable.IndexOf('=') is var index && index > 0 ? variable.Substring(0, index) : variable,
```

Comment:
are ranges available in this project?

Reply:
I don't think so (the refactoring isn't presented).